cpp/mrc/src/public/node/edge_registry.cpp

```diff
@@ -29,7 +31,7 @@ namespace mrc::node {
 
 void EdgeRegistry::register_converter(std::type_index writer_type, std::type_index reader_type, build_fn_t converter)
 {
-    VLOG(2) << "Registering converter for " << writer_type.hash_code() << " " << reader_type.hash_code();
+    // VLOG(2) << "Registering converter for " << type_name(writer_type) << " " << type_name(reader_type);
```

Comment:
Can we just change the VLOG level here, if its too verbose, instead of commenting this out.